lib/ansible/plugins/doc_fragments/template_common.py

```diff
@@ -69,6 +69,18 @@ class ModuleDocFragment(object):
     type: str
     default: '}}'
     version_added: '2.4'
+  comment_start_string:
+    description:
+    - The string marking the beginning of a comment statement.
+    type: str
+    default: '{#'
+    version_added: '2.9'
```

Comment:
As of now, this would need to be 2.10

Reply:
Done.

Reply:
Thanks. The change is almose done. Do I need any extra work!?

Reply:
a changelog, in this case minor_change heading seems apropos

Reply:
@bcoca , thanks for your comment.